lcls-twincat-motion/Library/POUs/Motion/FB_EncSaveRestore.TcPOU

```diff
@@ -45,17 +49,35 @@ END_VAR
     IF fbSetPos.Error THEN
         // Keep the error latched, it can disappear if Execute is set to FALSE
         nLatchError := fbSetPos.ErrorID;
-        // Alert the user that something has gone wrong
-        stMotionStage.bError := TRUE;
-        stMotionStage.nErrorId := fbSetPos.ErrorID;
-        stMotionStage.sCustomErrorMessage := 'Error loading previously saved position.';
+        nCurrTries := nCurrTries + 1;
+        IF nCurrTries >= nMaxRetries THEN
+            // Alert the user that something has gone wrong
+            stMotionStage.bError := TRUE;
+            stMotionStage.nErrorId := nLatchError;
+            stMotionStage.sCustomErrorMessage := 'Error loading previously saved position.';
```

Comment:
I wonder if this is worth a centralized log message so we can determine how prevalent this is. I'm leaning toward "no" as the patch seems likely sufficient.

Reply:
I'm 50/50 on this, I expect this to fix everything but I also would prefer to notice and fix it sooner next time if it breaks again somehow.

Taking a step back, should all stMotionStage errors become a centralized log message? Well, that might be redundant if the error is also an NC error. There's time to think, I won't deploy this fix anywhere until the PAMM.

Reply:
That's a good point, and I agree that it would be appropriate.

It reminds me of what @ghalym experienced the other day with ADS-based polling being insufficient to pick up on an error that only occurred during a single PLC cycle, whereas the asynchronously-emitted log message made its way through without issue.

And that reminds me - we might want to set all error messages such as this to 'monitor' instead of poll on the pragma side...

Reply:
https://github.com/pcdshub/lcls-twincat-motion/pull/140